Examples/UICatalog/Scenarios/Bars.cs

```diff
@@ -523,18 +523,18 @@ public void ConfigStatusBar (Bar bar)
 
         bar.Add (shortcut);
 
-        var button1 = new Button
+        var Button1 = new Button
```

Comment:
A local variable should be start with a lower letter. Should be as before `button1`. I think you rather wanted do `Button button1 = new ()`.